_sass/blocks/revenues/_revenues_steps.scss

```diff
@@ -5,38 +5,60 @@
 // Styleguide blocks.revenues_steps
 
 .revenues_steps-heading {
-	@include heading(h1);
+
+  @include respond-to(medium-down) {
+
+    @include heading(h4);
+
+  }
+
+  @include respond-to(tiny-down) {
+
+    @include heading(h5);
+
+    font-weight: $weight-light;
+  }
+
+  @include heading(h1);
```

Comment:
Expected item on line 22 to appear before line 9. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets